common.blocks/page/_view/page_view_500.bemtree.js

```diff
@@ -0,0 +1,19 @@
+block('page').mod('view', '500').content()(function() {
```

Comment:
It's more about having some BEM entity to apply templates to.
If you have a block or element it's possible to create template for it. But here we don't have any so it's fine to keep it like this.